driver-async/src/main/com/mongodb/async/client/MongoIterableImpl.java

```diff
@@ -69,6 +69,7 @@ ReadConcern getReadConcern() {
         return readConcern;
     }
 
+    @Override
```

Comment:
Since FindIterableImpl stores its batchSize in FindOptions, it also has to override this method.

Interesting that Coverity flagged FindIterableImpl.batchSize(int batchSize) as dangerous, as its the only subclass that doesn't call super.batchSize(batchSize).  Perhaps we should consider refactoring.

Reply:
Fixed. Yes, worth a ticket - will fix post opertations refactor.